### PR TITLE
Added build script for react-zoom-pan-pinch

### PR DESCRIPTION
**Pull request by Dhananjay-Sathe**

### CODE REVIEW

**Review by gerrith3 — COMMENTED** [review]
It seems like the build script should take a PACKAGE_VERSION as an argument; if no argument supplied, use the one this was tested against.  That would let someone use the same script against newer versions more easily.

**Comment by seth-priya** [comment]
@amitsadaphule

**Comment by seth-priya** [comment]
@amitsadaphule please review

File: r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#                 package and/or distribution. In such case, please
+#                 contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_VERSION=${1:-f78c546}
+
+#Install all dependencies.
+sudo yum clean all
+sudo yum -y update
```
Comment (amitsadaphule) [inline r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh:23]: Are these really necessary? If not, please remove.

File: r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+source ~/.nvm/nvm.sh
+
+#Install node version v12.19.1
+if [ `nvm list | grep -c "v12.19.1"` -eq 0 ]
+then
+        nvm install v12.19.1
+fi
+
+        nvm alias default v12.19.1
```
Comment (amitsadaphule) [inline r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh:40]: Please fix the indentation.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]
Other than the 2 changes suggested, rest looks OK to me. @Dhananjay-Sathe, please set your name and email in git config before push. The VM seems to have Sudeep's id, name in git config.

File: r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh (hunk 1 above)
Comment (Dhananjay-Sathe) [reply r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh:23]: @amitsadaphule These are not necessary. I removed it.

File: r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh (hunk 2 above)
Comment (Dhananjay-Sathe) [reply r/react-zoom-pan-pinch/react_zoom_pan_pinch_rhel_7.6.sh:40]: Fixed it.

**Comment by amitsadaphule** [comment]
LGTM

**Pull request merged**